client/pages/search-result.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import StarRating from '../components/star-rating';
+
+class SearchResultPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      chefs: []
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  componentDidMount() {
+    fetch(`/api/search/${this.props.selectedCuisine}`)
+      .then(res => res.json())
+      .then(data => {
+        this.setState({ chefs: data });
+      });
+  }
+
+  handleClick(event) {
+    const chefid = event.target.getAttribute('chefid');
+    window.location.hash = 'chefProfile?chefId=' + chefid;
+  }
```

Comment:
Instead of a click handler on a button, use an anchor tag. you can style it like a button if you want.